src/archivist/services/documentTypes.json

```diff
@@ -1,191 +1,198 @@
+    }
+  },
+  "Human Rights Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "shareholders",
+      "object": "human rights"
+    }
+  },
+  "In-App Purchases Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "in-app and other virtual goods purchase usage"
+    }
+  },
+  "Review Guidelines": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "developer",
+      "object": "review process of provided content"
+    }
+  },
+  "Brand Guidelines": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "developer",
+      "object": "usage of the service provider’s brand"
+    }
+  },
+  "Quality Guidelines": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "quality of content produced by means of the service"
+    }
+  },
+  "Data Controller Agreement": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "data controllers (in the sense of GDPR)",
+      "object": "end user personal data"
+    }
+  },
+  "Data Processor Agreement": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "data controllers (in the sense of GDPR)",
+      "object": "status of data processor (in the sense of GDPR) for the service provider"
+    }
+  },
+  "User Consent Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "data controllers (in the sense of GDPR)",
+      "object": "user consent collection and storage"
+    }
+  },
+  "Closed Captioning Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "closed captioning on streamed content"
+    }
+  },
+  "Seller Warranty": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "protection of end users that act as sellers"
+    }
+  },
+  "Single Sign-On Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "connecting user account to other services"
+    }
+  },
+  "Vulnerability Disclosure Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "vulnerability reporter",
+      "object": "how to report a security vulnerability or issue"
+    }
+  },
+  "Live Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "end user’s live service usage"
+    }
+  },
+  "Ad Publishing Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "acceptable and unacceptable usage"
+    }
+  },
+  "Complaints Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "anyone",
+      "object": "how generic complaints will be handled"
+    }
+  },
+  "Transport Policy": {
```

Comment:
Based on some [example airline](https://www.transavia.com/en-EU/conditions/conditions-of-carriage/) website, I found that “Conditions de transport” seem to be “Conditions of Carriage” in English.
When I look this term up in a search engine, I find indeed a [definition](https://www.icao.int/sustainability/Documents/ConsumerInterests_B.pdf) for this term by the ICAO (UN agency for civil aviation), which seems very fit as a source:

>  “Conditions of carriage” mean the terms and conditions established by an air carrier in respect of its carriage, and are filed as applicable tariff rules.

On Wikipedia, I find, for [rail travel](https://en.wikipedia.org/wiki/CIV_(rail_travel)):

> The full current title is the Uniform Rules concerning the Contract of International Carriage of Passengers by Rail (CGT-CIV, [French](https://en.wikipedia.org/wiki/French_language): Conditions générales de transport pour le transport international ferroviaire des voyageurs), formerly the Uniform Rules concerning the Contract for International Carriage of Passengers and Luggage by Rail.

I thus suggest:

```suggestion
  "Conditions of Carriage": {
```

Reply:
👍 ✅